gateway-api/src/lib.rs

```diff
@@ -69,8 +69,7 @@ mod tests {
                     .metadata
                     .name
                     .ok_or(Error::msg("could not find GatewayClass name"))?,
-                addresses: None,
-                listeners: vec![],
+                ..Default::default()
```

Comment:
Changed the test to use `Default::default` so it won't break next time someone else updates versions.